src/app/year-calendar/year-calendar-config.ts

```diff
@@ -0,0 +1,28 @@
+import {locale, Moment} from 'moment';
+import {ICalendar, ICalendarInternal} from '../common/models/calendar.model';
+import {ECalendarValue} from '../common/types/calendar-value-enum';
+
+export interface IConfig {
+  isYearDisabledCallback?: (date: Moment) => boolean;
+  allowMultiSelect?: boolean;
+  yearFormat?: string;
+  yearFormatter?: (month: Moment) => string;
+  format?: string;
+  isNavHeaderBtnClickable?: boolean;
```

Comment:
what will this do?

Reply:
Right -- I'll omit this, the formatting parts. Year can only be year/YYYY, as I'm sure is your point.  The year calendar is molded from the month calendar, at first very closely to get it running. This detailed feedback from you is precisely what I was hoping for, to help me finish the code.

Reply:
s

Reply:
Looking at this now, and I'm not sure what, specifically, you're refering to. As far as I can see, the Year Calendar Config needs to specify a year format string, even if it can't be anything else than `YYYY`. However, it seems I can do without the `yearFormatter`. Can you please elaborate?